### PR TITLE
Align panel heights

### LINKED ISSUE #188: Consistent panel height
All of the panels below the map should have the same height. If this forces a panel to be larger than its content, then the content should be aligned to the top of the panel.

### PR DESCRIPTION
**Pull request by FloofyPancakes**
Closes #188 
Aligned the bottom of the panels

image

### CODE REVIEW

File: src/components/Schedule.vue [hunk 1]
```diff
@@ -104,4 +104,7 @@ ul {
 .sunday-times {
   margin-left: 23.5px;
 }
+.panel-height {
+  margin-bottom: 59px;
```
Comment (Isaac-the-Man) [inline src/components/Schedule.vue:108]: Don't hard code margin

**Review by Isaac-the-Man — CHANGES_REQUESTED** [review]
Huge blank space in mobile mode.
image

**Comment by Gerzer** [comment]
> Huge blank space in mobile mode.
> image

Is there a way to keep the blank space when the cards are displayed side-by-side but to remove it when they’re stacked (like on mobile)?

**Comment by Isaac-the-Man** [comment]
> > Huge blank space in mobile mode.
> > image
> 
> Is there a way to keep the blank space when the cards are displayed side-by-side but to remove it when they’re stacked (like on mobile)?

There should be a way to do that, I'll take a look.

**Comment by Isaac-the-Man** [comment]
Fixed panel height on mobile devices.

image

image

**Pull request merged**